todo_app/app.py

```diff
@@ -1,15 +1,24 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items, add_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/index')
 def index():
-    return 'Hello World!'
+	todolists = get_items()
```

Comment:
"todolists" is a slightly misleading name - it's actually a single to do list of many to do items